src/Components/Common/LabelForm/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Heading from '@/Components/Heading';
+import { ErrorMsg } from '@/Pages/CreateRecruitment/page';
+import { RecruitFormWithSelect } from '@/Types/study';
+import { FieldErrors } from 'react-hook-form';
+import styled from 'styled-components';
+
+export interface LabelFormProps<T> {
+  /** 라벨 */
+  label?: string;
+
+  /** 자식 노드 (폼) */
+  children: React.ReactNode;
+
+  /** RHF와 호환되는 에러 객체 */
+  errors?: FieldErrors<T>;
+
+  /** RHF 필드 키 */
+  name?: keyof T;
+}
+
+export const LabelForm = ({ label, children, name, errors }: LabelFormProps<RecruitFormWithSelect>) => {
```

Comment:
```suggestion
export const LabelForm = <T extends FieldValues = FieldValues>({ label, children, name, errors }: LabelFormProps<T>) => {
```

`LabelForm`을 제네릭하게 만듭니다.